packages/nc-gui/pages/index/index/index.vue

```diff
@@ -32,6 +33,8 @@ const filterQuery = ref('')
 
 const projects = ref<ProjectType[]>()
 
+const { appInfo } = $(useGlobal())
```

Comment:
It's not an error but unnecessary wrap in `$()`.
`$()` will unwrap the value but if you're just using in the template, it will be unwrapped regardless of `$()` or not.